tox.ini

```diff
@@ -52,12 +52,14 @@ deps =
     xdist: pytest-xdist
     cov: pytest-cov
     jwst: jwst[test] @ git+https://github.com/spacetelescope/jwst.git
+use_develop=
+    cov: true
```

Comment:
would this also work with `package`? 
```ini
package: 
    cov: editable
    !cov: wheel
```

Reply:
That does appear to work. Is that preferable?

Reply:
The suggested `package` option seems more readable. I switched it to that and the coverage job passed (with similar coverage, differences attributable to files omitted from coverage https://github.com/spacetelescope/stdatamodels/pull/138#issuecomment-1454028292)